Tests/RxCocoaTests/DelegateProxyTest.swift

```diff
@@ -56,6 +56,70 @@ extension TestDelegateControl {
 // MARK: Tests
 
 final class DelegateProxyTest : RxTest {
+    
+    static var extendDelegateProxy: Void = {
```

Comment:
I'm little bit confusing, we should call `DelegateProxy.extendProxy` in runtime, but it should be only one time.
`static var` is always lazy property and I use this one in `setup` function.
Is it better that adding `main` function in test? `dispatch_once_t` was depricated on Swift 3.0.